backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -1545,39 +1536,46 @@ trait StandardAsyncExecutionActor
     serviceRegistryActor.putMetadata(jobDescriptor.workflowDescriptor.id, Option(jobDescriptor.key), metadataKeyValues)
   }
 
-  def tellBard(state: StandardAsyncRunState): Unit =
-    getStartAndEndTimes(state) match {
-      case Some(startAndEndTimes: StartAndEndTimes) =>
-        val dockerImage =
-          RuntimeAttributesValidation.extractOption(DockerValidation.instance, validatedRuntimeAttributes)
-        val cpus = RuntimeAttributesValidation.extract(CpuValidation.instance, validatedRuntimeAttributes).value
-        val memory = RuntimeAttributesValidation
-          .extract(MemoryValidation.instance(), validatedRuntimeAttributes)
-          .to(MemoryUnit.Bytes)
-          .amount
-        serviceRegistryActor ! BardEventRequest(
-          TaskSummaryEvent(
-            workflowDescriptor.id.id,
-            workflowDescriptor.possibleParentWorkflowId.map(_.id),
-            workflowDescriptor.rootWorkflowId.id,
-            jobDescriptor.key.tag,
-            jobDescriptor.key.call.fullyQualifiedName,
-            jobDescriptor.key.index,
-            jobDescriptor.key.attempt,
-            state.getClass.getSimpleName,
-            platform.map(_.runtimeKey),
-            dockerImage,
-            cpus,
-            memory,
-            startAndEndTimes.jobStart.toString,
-            startAndEndTimes.cpuStart.map(_.toString),
-            startAndEndTimes.jobEnd.toString,
-            startAndEndTimes.jobStart.until(startAndEndTimes.jobEnd, ChronoUnit.SECONDS),
-            startAndEndTimes.cpuStart.map(_.until(startAndEndTimes.jobEnd, ChronoUnit.SECONDS))
-          )
-        )
-      case _ => ()
-    }
+  /**
+   * Reports metrics to bard.
+   * @param jobStart: Time that Cromwell started processing this job. Should be the earliest recorded time that this job has done...anything.
+   * @param vmStartTime: Time that the backend started spending money.
+   * @param vmEndTime: Time that the backend stopped spending money.
+   */
+  def tellBard(terminalStateName: String,
```

Comment:
The only thing different about this function is that the start and end times are provided as parameters, rather than being fetched within the function body.